SPEC.md

```diff
@@ -106,6 +127,12 @@ Success is indicated by a return code of zero and the following JSON printed to
 The result is returned in the same format as specified in the [configuration](#network-configuration).
 The specification does not declare how this information must be processed by CNI consumers.
 Examples include generating an `/etc/resolv.conf` file to be injected into the container filesystem or running a DNS forwarder on the host.
+The `interfaces` describes specific network interfaces the plugin created and their IP configuration.
+If the plugin does not use L2 addressing and thus MAC addresses are not meaningful, then an interface element's "mac" key may be omitted.
+The `sandbox` key must be provided for interfaces created or moved into a sandbox like a network namespace or a hypervisor/VM.  For container/namespace-based environments the `sandbox` key should be the full filesystem path to the network namespace of that sandbox. Hypervisor/VM-based plugins should return an ID unique to the virtualized sandbox the interface was created in.
```

Comment:
The last word should probably be "for" instead of "in" since CNI will probably not actually act inside a VM but just prepare interfaces for it.